Pages/cours.html

```diff
@@ -51,21 +52,30 @@ <h3 class="text-center">Administrer les cours</h3>
                             <input type="text" class="form-control" id="teacherInput" placeholder="Prof">
                         </td>
                         <td>
-                            <textarea class="form-control" id="descriptionInput" rows="5" cols="33"></textarea>
+                            <textarea class="form-control" id="descriptionInput" rows="3" cols="25"></textarea>
                         </td>
                         <td>
-                          <button type="button" class="btn btn-success text-uppercase text-center" onClick="postClass()"> ajouter </button></td>
+                          <button type="button" class="btn btn-success text-uppercase text-center" onClick="createClass()"> ajouter </button></td>
                       </tr>
                   </form>
                 </tbody>
               </table>
               <div class="bg-white">
-                <button type="button" class="btn btn-outline-danger btn-lg btn-block text-uppercase text-center " 
-                        id="deleteAllBtn" onClick="deleteAll(false)"> tous suppimer 
-                </button>
+                  <div class="row p-2">
+                      <div class="col">
+                        <button type="button" class="btn-delete btn-outline-danger btn-lg btn-block text-uppercase text-center " 
+                            id="deleteAllBtn" 
+                            onClick="deleteAll(false)"> tous suppimer 
+                        </button>
+                      </div>
+                      <div id="initializeBtn" class="col-initialize col-6">
```

Comment:
You should set the id on the button not the div and the name should be reinitBtn (initialization happends the first time)

Reply:
Done!